### ISSUE TITLE
Components do not render children

### ISSUE THREAD

**Issue by bdchauvette** [StephanHoyer/mithril-node-render#49]
Components that have children seem do not actually render their children.

I've created [a gist demonstrating the behavior](https://gist.github.com/bdchauvette/bc9205a33538e025f1e503dc0211b171).

Here is the output I get when I run ospec on the code from the gist:

```
parent component:
"<div></div>"
should equal
"<div>howdy</div>"

    at o (C:\Users\Ben\Code\bc9205a33538e025f1e503dc0211b171\index.js:24:50)


parent component:
"<div></div>"
should equal
"<div><span>howdy</span></div>"

    at o (C:\Users\Ben\Code\bc9205a33538e025f1e503dc0211b171\index.js:25:61)


4 assertions completed in 24ms, of which 2 failed
```

It seems like adding `children: copy(view.children)` to the component vnode creation fixes things, but I'm not sure if that's the proper fix?

**Comment by StephanHoyer** [comment]
I would assume this is a proper fix

Could you provide a PR with test plz?

Am 13.12.2016 18:48 schrieb "Ben Chauvette" <notifications@github.com>:

**Comment by StephanHoyer** [comment]
fixed with #50

**Comment by StephanHoyer** [comment]
fyi just released mithril-node-render@1.0.1-4. Now we also support component child rendering

**Comment by bdchauvette** [comment]
Awesome, thanks! :+1:
